src/cudamatrix/cu-vector-test.cc

```diff
@@ -345,16 +345,7 @@ template<typename Real> void CuVectorUnitTestSum() {
     A.SetRandn();
     ones.Set(1.0);
 
-    Real x = VecVec(A, ones);
-    Real y = A.Sum();
```

Comment:
Phis and a couple of other files seem to have some changes that are reversions of recent changes to master.  Possibly a sign of some weirdness in your use of git?